inspirehep/ext/jinja_filters/record.py

```diff
@@ -407,3 +418,14 @@ def ads_links(record):
         else:
             link = "%sauthor=%s" % (ADSURL, record['name']['preferred_name'])
         return link
+
+    @app.template_filter
+    def citation_count(record):
```

Comment:
Is this filter being used?

It seems like you are accessing directly the value in your templates without the need for a filter.